src/components/DpUploadFiles/DpUpload.vue

```diff
@@ -242,7 +242,7 @@ export default {
 
     this.uppy.on('upload-error', (file, error, response) => {
       console.error(error)
-      dplan.notify.error(Translator.trans('error.fileupload'))
+      dplan.notify.error(de().strings.errorFileUpload)
```

Comment:
can't we move the `const local` which is set  in the initialize-method move to `data` and then use it here like `this.local.errorFileUpload` ? 
(maybe You find a better name then `local`)

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/918/commits/02678d74fdfd795490a0149f7494e48f350ffa73